### PR TITLE
Transform code to gem

### PR DESCRIPTION
**Pull request by faloi**
Refs uqbar-project/mumuki-gobstones-server#24

### CODE REVIEW

**Comment by flbulgarelli** [comment]
I think it good be a good idea not to include gobstones as a submodule here. That way anyone can use   this gem with any gobstones implementation, right?

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2015 Franco Leonardo Bulgarelli
```
Comment (flbulgarelli) [inline LICENSE.txt]: And @faloi  too :smile:

File: gobstones-spec.gemspec [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'gobstones/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'gobstones-spec'
+  spec.version       = Gobstones::VERSION
+  spec.authors       = ['Federico Aloi']
+  spec.email         = ['federico.aloi@gmail.com']
```
Comment (flbulgarelli) [inline gobstones-spec.gemspec]: Haha, you have not being consistent here regarding attributions :stuck_out_tongue_closed_eyes:

Comment (faloi) [inline gobstones-spec.gemspec]: The risks of copy&paste

**Comment by faloi** [comment]
> @flbulgarelli: I think it good be a good idea not to include gobstones as a submodule here. That way anyone can use   this gem with any gobstones implementation, right?

Mmmm, it could be a good idea but right now it heavily depends on my fork of Gobstones. I'll address it later: https://github.com/uqbar-project/gobstones-spec/issues/2

**Pull request merged**